apps/ios/LiveVideo/LiveVideo/Views/Participants/ViewersViewModel.swift

```diff
@@ -0,0 +1,114 @@
+//
+//  Copyright (C) 2021 Twilio, Inc.
+//
+
+import Combine
+
+class ViewersViewModel: ObservableObject {
+    struct Viewer: Identifiable {
+        let userIdentity: String
+        var id: String { userIdentity }
+     
+        init(viewer: ViewersStore.Viewer) {
+            userIdentity = viewer.userIdentity
+        }
+        
+        init(raisedHand: RaisedHandsStore.RaisedHand) {
+            userIdentity = raisedHand.userIdentity
+        }
+    }
+    
+    @Published var viewersWithRaisedHand: [Viewer] = []
+    @Published var viewersWithoutRaisedHand: [Viewer] = []
```

Comment:
The UX design calls for displaying all the viewers in a list with the raised hands at the top.

Long story short we must use separate sync maps for viewers and raised hands because of the way sync works.

The viewers map contains all viewers including viewers that may have their hand raised. And so the client must filter users out of the viewers map if they have their hand raised. Why not have the server do this filtering?

1. It seemed more general purpose and UI agnostic to simply have a set of all viewers and then the client can do whatever they want with it.
2. We would have to rename the viewers map to something like "viewers without hand raised" which is fine but a bit clunky.
3. If the server did the filtering the client would not be able to animate list updates in the UI because the updates to the viewers map and raised hands map would come in at different times. And even worse we would see strange things like a user showing up twice or not at all momentarily. In my next ticket we will see issues like this when the user switches between viewers and speakers but I think that won't be bad because the user is switching between different list sections in the UI as opposed to changing position inside one section. It seems like the latter would look like a bug if not done well. Or is it just me?

Make sense? Thoughts?